src/Monitor/Agents.vue

```diff
@@ -2,105 +2,24 @@
 <div>
   <toggleBar></toggleBar>
   <b-collapse v-model="showCollapse" id="collapseAgentManager" class="mt-2">
-
-    <div class="row toggle-bar-custom">
-      <div class="title">Agent States</div>
-    </div>
-
-    <b-table style="margin-top:10px" small striped hover
-      :items="computedAgentsStats"
-      :fields="fieldsStats">
-    </b-table>
-
-    <div class="row toggle-bar-custom">
-      <div class="title">Agents</div>
-    </div>
-    <div class="row">
-      <div class="col-2">
-        <div class="row toggle-bar-custom">
-          <div class="title">Filter</div>
-        </div>
-        <b-form-input size="sm" v-model="filter" placeholder="Search..." style="margin-top:10px" />
-        <b-form-select size="sm" v-model="selectedProfile" style="margin-top:10px">
-          <option v-for="group in this.groups" :value=group.name>{{group.name}}</option>
-        </b-form-select>
-        <b-form-select size="sm" v-model="selectedCustomer" style="margin-top:10px">
-          <option v-for="client in this.clients" :value=client.name>{{client.name}}</option>
-        </b-form-select>
-        <b-form-select size="sm" v-model="selectedState" style="margin-top:10px">
-          <option v-for="state in this.states" :value=state.name>{{state.name}}</option>
-        </b-form-select>
-      </div>
-      <div class="col-10">
-        <b-table style="margin-top:10px" small striped hover
-          :items="computedAgents"
-          :fields="fields"
-          :filter="filter"
-          :sort-by="sortBy"
-          :sort-desc="sortDesc"
-          @sort-changed="onSortingChanged">
-          <template slot="agent_detail" slot-scope="data">
-            <div class="agent-state-text">{{data.item.agent_name + ' ' + data.item.agent_login}}</div>
-            <div class="agent-state-text"><b>Profile: </b>{{data.item.agent_group}}</div>
-            <div class="agent-state-text"><b>Phone: </b>{{data.item.agent_phone}}</div>
-            <div class="agent-state-text"><b>Skills: </b>{{data.item.agent_skills}}</div>
-            <div class="agent-state-text"><b>Customer: </b>{{data.item.agent_client}}</div>
-          </template>
-          <template slot="actions" slot-scope="data">
-            <b-button v-if="data.item.state == 'release'" size="sm" variant="warning" @click="available(data.item)">Available</b-button>
-            <b-button v-else size="sm" variant="primary" @click="release(data.item)">Release</b-button>
-            <b-button size="sm" variant="danger" @click="stop(data.item)">Kill</b-button>
-          </template>
-        </b-table>
-      </div>
-    </div>
+    <agentProfiles :agents="agents" :groups="groups"></agentProfiles>
```

Comment:
I would prefer tags in kebab-style, e.g. agent-profiles, see bootstrap-vue namings, for the sake of uniformity.

Reply:
changed to use kebab-style for tags